server/__init__.py

```diff
@@ -45,7 +46,8 @@
     'games',
     'control',
     'abc',
-    'protocol'
+    'protocol',
+    'TeamMatchmakingService'
```

Comment:
Can we alphabetize these, and keep all the stuff with capital letters next to each other?